VMWare/vmware-vcenter/_meta/fields.yml

```diff
@@ -0,0 +1,64 @@
+vmware_vcenter.ip_address:
+  name: vmware_vcenter.ip_address
+  description: "The IP address of the VMware vCenter server."
+  type: keyword
```

Comment:
According to the usage of this field, I think we can replace its usage with `host.id` and remove it from the taxomony.